packages/screenshoter/src/take-screenshot.js

```diff
@@ -63,7 +63,9 @@ async function takeScreenshot({
     if (!window && !driver.isNative) {
       await scrollIntoViewport({context: target.context, region: target.region, scroller: target.scroller, logger})
     }
-
+    if (fully && !region && target.scroller) {
+      await target.scroller.moveTo({x: 0, y: 0})
+    }
```

Comment:
@tourdedave 
```suggestion
    if (fully && !target.region && target.scroller) {
      await target.scroller.moveTo({x: 0, y: 0})
    }
```

I also this that if you scrolled to 0,0 here we don't need to scroll to 0,0 again in `takeStitchedScreenshot`.

Reply:
I'm not sure if it doesn't make sense. It's hard for me to reason about. It would require a lot of re-testing to verify.